### ISSUE TITLE
[TF 2.0 API Docs] tf.keras.activations.tanh

### ISSUE THREAD

**Issue by rohanmeh** [tensorflow/tensorflow#25827]
**System information**
- TensorFlow version: 2.0
- Doc Link: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/tanh
* **Correct Links** https://github.com/tensorflow/tensorflow/blob/master/python/keras/activations.py is incorrect. The correct link should be https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py.
* **Clear Description**
  No description of what the symbol does or what happens when arguments are passed to the symbol.
* **Usage Example**
  No usage example is provided.
* **Parameters Defined**
  Parameters are not defined.
* **Returns Defined**
  Returns are not defined.
* **Raises Listed and Defined**
  Errors are not defined.

**Comment by shashvatshahi1998** [comment]
I modified tanh(),sigmoid() and exp() functions

**Comment by dhirensr** [comment]
@dynamicwebpaige : could you point the source code of the docs_src? 
i could not find the source code for this docs at  https://github.com/tensorflow/docs/ 
i could add the docs if pointed out the file

**Comment by jvishnuvardhan** [comment]
Automatically closing this out since I understand it to be resolved by the PR #29725 (merged already), but please let me know if I'm mistaken.Thanks!
